### PR TITLE
CApplication: move StartServer method to CNetworkServices

### PR DESCRIPTION
**Pull request by lrusak**
This method has no business in CApplication so let's move it to where it's relevant.

I've also cleaned up the method a bit and remove the unused bWait param

@enen92 for the python and doxygen changes

### CODE REVIEW

**Review by fuzzard — APPROVED** [review]

File: xbmc/interfaces/legacy/ModuleXbmc.h [hunk 1]
```diff
@@ -744,10 +744,9 @@ namespace XBMCAddon
     /// | xbmc.SERVER_EVENTSERVER   | [Set eventServer part that accepts remote device input on all platforms](http://kodi.wiki/view/EventServer)
     /// | xbmc.SERVER_ZEROCONF      | [Control Kodi's Avahi Zeroconf](http://kodi.wiki/view/Zeroconf)
     /// @param bStart               bool - start (True) or stop (False) a server
-    /// @param bWait                [opt] bool - wait on stop before returning (not supported by all servers)
     /// @return                     bool - True or False
     ///
-    ///
+    /// @python_v20 Removed option **bWait**.
```
Comment (enen92) [inline xbmc/interfaces/legacy/ModuleXbmc.h:749]: Please keep the 2 empty lines (as before) and move this new line below the thematic break (---). Keep an empty line before the example (as before). Yeah it sounds silly but doxygen breaks easily :)
Thanks for the improvement!

Comment (lrusak) [reply xbmc/interfaces/legacy/ModuleXbmc.h]: Updated thanks!

**Review by enen92 — APPROVED** [review]

**Comment by lrusak** [comment]
jenkins build this please

**Comment by lrusak** [comment]
jenkins build this please

**Pull request merged**